commit 5022c3393d032eedf5cbc4e7def6fea760f19127
Author: Isuru Fernando <isuruf@gmail.com>
Date:   Sun Sep 23 16:21:30 2018 -0500

    Fix strcasecmp incorrect detection due to warning about implicit declaration

diff --git a/config/hwloc.m4 b/config/hwloc.m4
index 117301f5..bb55349f 100644
--- a/config/hwloc.m4
+++ b/config/hwloc.m4
@@ -491,7 +491,7 @@ EOF])
     # Needed for Windows in private/misc.h
     AC_CHECK_TYPES([ssize_t])
     AC_CHECK_DECLS([snprintf], [], [], [AC_INCLUDES_DEFAULT])
-    AC_CHECK_DECLS([strcasecmp], [], [], [AC_INCLUDES_DEFAULT])
+    AC_CHECK_DECLS([strcasecmp], [], [], [[#include <strings.h>]])
     # strdup and putenv are declared in windows headers but marked deprecated
     AC_CHECK_DECLS([_strdup], [], [], [AC_INCLUDES_DEFAULT])
     AC_CHECK_DECLS([_putenv], [], [], [AC_INCLUDES_DEFAULT])
